config.yml

```diff
@@ -42,7 +42,7 @@ defaults:
 
   statelessHostname:
     enabled: false
-    secret: process.env.DNS_SERVER_SECRET
+    secret: !env DNS_SERVER_SECRET
```

Comment:
i'm not familiar with this syntax so getting a review from someone else for this line would be great

Reply:
To be honest, we could just drop this, as nothing ever provides the env var (or if it did, it didn't work).  This is at least less-awfully-broken than the previous :)